lms/templates/navigation.html

```diff
@@ -107,7 +107,7 @@ <h1 class="logo">
     </ol>
 
     <ol class="right nav-courseware">
-      % if not settings.FEATURES['DISABLE_LOGIN_BUTTON']:
+      % if not settings.DISABLE_REGISTER_BUTTON:
```

Comment:
You could think of it as a bug. The DISABLE_LOGIN_BUTTON setting was controlling visibility of the Register and Login buttons. We wanted finer control; remove only the Register button.